webviz_subsurface/plugins/_volumetric_analysis/controllers/fipfile_qc_controller.py

```diff
@@ -0,0 +1,118 @@
+from typing import Callable
+import pandas as pd
+import dash
+from dash.dependencies import Input, Output
+from dash.exceptions import PreventUpdate
+import dash_html_components as html
+import plotly.express as px
+import plotly.graph_objects as go
+import webviz_core_components as wcc
+from ..utils.table_and_figure_utils import create_table_columns, create_data_table
```

Comment:
```suggestion
from typing import Callable

import pandas as pd
import dash
from dash.dependencies import Input, Output
from dash.exceptions import PreventUpdate
import dash_html_components as html
import plotly.express as px
import plotly.graph_objects as go
import webviz_core_components as wcc

from ..utils.table_and_figure_utils import create_table_columns, create_data_table
```

Reply:
Also use `from dash import html` instead of `import dash_html_components as html`